rqt_py_common/src/rqt_py_common/topic_helpers_from_ros_com.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (c) 2011, Dorian Scholz, TU Darmstadt
```

Comment:
Is this file used? If it is, let me know so I can review it. There are some things I'd change

Reply:
It is not currently used. It is a direct port of the functionality in ROS Com for ros1. I ported this thinking that it would enable plotting bounded and unbounded arrays with rqt_plot but it wasn't the issue. Ultimately, I thought this may still be useful at some point so I didn't want to remove it.  (the msgeval stuff is really cool)

I will move this to a new PR.